extension/js/common/lang.ts

```diff
@@ -2,11 +2,15 @@
 
 'use strict';
 
+import { FLAVOR } from './core/const.js';
+
 /* eslint-disable max-len */
 
+const isEnterpriseBuildUsed = FLAVOR === 'enterprise';
+
 const contactMinimalSubsentence = (isFesUsed: boolean) => isFesUsed ? 'contact your Help Desk' : 'write us at human@flowcrypt.com';
```

Comment:
Here since it's the base method actually doing the work, `isFesUsed || isEnterpriseBuildUsed` would be appropriate.

It's always best to put the needed conditions in whatever method is the "choke point" of the implementation. A choke point is the place that will always be executed. In this implementation, `contactMinimalSubsentence` and `contactForSupportSubsentence` are the choke points, so you can put the logic there, and then it's not needed elsewhere.

Reply:
Thank you - I will keep this and the other helpful comments in mind.